### ISSUE TITLE
TRTL101 Bugs, fixes, etc.

### ISSUE THREAD

**Issue by RocksteadyTC** [turtlecoin/meta#133]
Please be concise and keep this limited to posting issues if possible.

**Comment by zpalmtree** [comment]
The final question of the section 'Welcome to the core suite -> Miner` has an incorrect answer.
The correct answer should be 'Processor (CPU)'.
The answer that is marked correct is 'Graphics Card (GPU)'

**Comment by zpalmtree** [comment]
Upon completing all the questions in the last section, the 'Next' button is faded out. It is not immediately obvious this is the end of the course, perhaps there should be a 'Finish' button?

**Comment by Whocaresright** [comment]
Shouldn't the option to show correct answer be displayed when someone answers the question Incorrectly.

Right now, It's other way around.

https://ibb.co/zZXhK8Z
https://ibb.co/hRVxxSb

**Comment by rashedmyt** [comment]
Wrong information in TurtleCoin Community.. Yellow is for Testers and Brown is for Students

**Comment by rashedmyt** [comment]
formatting in Turtle-Service on line 3 of 2nd para

**Comment by rashedmyt** [comment]
extra unnecessary text in turtle-service last MCQ

**Comment by RocksteadyTC** [comment]
@Whocaresright can you elaborate? The first image link doesnt load. 
![image](https://user-images.githubusercontent.com/34389545/49772871-6beff180-fcb4-11e8-83a7-d129e137e6d2.png)

**Comment by RocksteadyTC** [comment]
@zpalmtree Please try now 
![image](https://user-images.githubusercontent.com/34389545/49773435-b6726d80-fcb6-11e8-9ccd-48cfeaa3d6a2.png)

**Comment by Sajo8** [comment]
http://edu.turtlecoin.lol/courses/course-v1:TurtleEDU+DEV100+2018_T3/about
http://edu.turtlecoin.lol/courses/course-v1:TurtleEDU+DEV101+2019_T1/about

Fix the staff descriptions

**Comment by ace2727** [comment]
Completed the "Introduction to TurtleCoin" course. Below are all of the sections with questions which cannot be answered using only the information provided in each section's text:

-Meet the Daemon:
*What is the command you type to check your sync progress?
*What is a special technique you can use to sync blocks faster?
*About how many blocks are added to the TurtleCoin blockcain every year? (also "blockchain" is misspelled here)
*When you want to see a list of functions you can use in the daemon, what command do you type?

-Zedwallet - The Best CLI Wallet™
*You might have noticed every TurtleCoin wallet address starts with TRTL but how many characters are in every TRTL wallet address?  (For this question are we meant to manually count our wallet address characters or should the information be found in the text which was read?)

If it is intended for the students to find these answers outside of the course then I think we should provide links. Otherwise the information should be included in the text being read by the students.

**Comment by Whocaresright** [comment]
> @Whocaresright can you elaborate? The first image link doesnt load.
> ![image](https://user-images.githubusercontent.com/34389545/49772871-6beff180-fcb4-11e8-83a7-d129e137e6d2.png)

What I meant was, When you gave the wrong answer there was no option given to see the correct answer.
But when you gave the correct answer, It was showing the option to show correct answer.

Hopefully this will load
https://prnt.sc/ltjdt9
https://prnt.sc/ltje3t

**Comment by 19morpheus80** [comment]
http://edu.turtlecoin.lol/courses/course-v1:TurtleEDU+TRTL101+2019/courseware/fd470f755834464d8504e9ad5c50cd6f/e928beef0f05428c86976ecb5c906d69/?child=first
"TurtleCoind - Your Connection to the TRTL Network"

None of the answers are given in the body text.

**Comment by 19morpheus80** [comment]
http://edu.turtlecoin.lol/courses/course-v1:TurtleEDU+TRTL101+2019/courseware/fd470f755834464d8504e9ad5c50cd6f/e928beef0f05428c86976ecb5c906d69/?child=first
"Zedwallet - The Best CLI Wallet™"

Zedwallet has lots of easy functions that you can use to make your life easier, like "optimize" **which** will take all of your little turtle nickels and dimes and turn them into solid coins so you can send more in a single transaction.

**Comment by 19morpheus80** [comment]
> Completed the "Introduction to TurtleCoin" course. Below are all of the sections with questions which cannot be answered using only the information provided in each section's text:
> 
> -Meet the Daemon:
> *What is the command you type to check your sync progress?
> *What is a special technique you can use to sync blocks faster?
> *About how many blocks are added to the TurtleCoin blockcain every year? (also "blockchain" is misspelled here)
> *When you want to see a list of functions you can use in the daemon, what command do you type?
> 
> -Zedwallet - The Best CLI Wallet™
> *You might have noticed every TurtleCoin wallet address starts with TRTL but how many characters are in every TRTL wallet address? (For this question are we meant to manually count our wallet address characters or should the information be found in the text which was read?)
> 
> If it is intended for the students to find these answers outside of the course then I think we should provide links. Otherwise the information should be included in the text being read by the students.

Part of learning should include doing things off your own back, and that includes finding out how to get a computer to count the length of a string without being spoon-fed the procedure.  There are numerous ways to do that.

**Comment by 19morpheus80** [comment]
http://edu.turtlecoin.lol/courses/course-v1:TurtleEDU+TRTL101+2019/courseware/fd470f755834464d8504e9ad5c50cd6f/e928beef0f05428c86976ecb5c906d69/?child=first
Turtle-Service

The purpose of Turtle-Service is to give access to <i>programs</i> who need to manage a wallet, as opposed to a <i>human</i>, who would use something like Zedwallet.

I think the italic HTML tag was replaced by <em>, although <i> should still work.  Just not here.

**Comment by ace2727** [comment]
> Part of learning should include doing things off your own back, and that includes finding out how to get a computer to count the length of a string without being spoon-fed the procedure. There are numerous ways to do that.

Agreed. Usually those activities are the most rewarding and memorable. Just was not sure if the questions were meant to be answered based on the text or done as a kind of research/homework. Removing that ambiguity could be helpful. The same issue was raised in the following post.

> http://edu.turtlecoin.lol/courses/course-v1:TurtleEDU+TRTL101+2019/courseware/fd470f755834464d8504e9ad5c50cd6f/e928beef0f05428c86976ecb5c906d69/?child=first
"TurtleCoind - Your Connection to the TRTL Network"
None of the answers are given in the body text.

**Comment by RocksteadyTC** [comment]
I intended for every question asked to have an answer in the body of the text. This should be the case in every course, as url's do change.
